internal/common/splunk/common.go

```diff
@@ -17,8 +17,11 @@ package splunk
 import "strings"
 
 const (
+	HostnameLabel        = "host.hostname"
```

Comment:
I just realized these are the same as in this file https://github.com/open-telemetry/opentelemetry-collector/blob/master/translator/conventions/opentelemetry.go#L20.  If they are actually referring to the same thing then you can just refer to that package instead of redefining them here.  Looks like `source.type` isn't there but the other two are.